pennylane_lightning/experimental/_serialize.py

```diff
@@ -0,0 +1,219 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Helper functions for serializing quantum tapes.
+"""
+from typing import List, Tuple
+
+import numpy as np
+from pennylane import (
+    BasisState,
+    Hadamard,
+    Hamiltonian,
+    QubitStateVector,
+    Rot,
+)
+from pennylane.grouping import is_pauli_word
```

Comment:
```suggestion
from pennylane.pauli import is_pauli_word
```
grouping is deprecated